mobile/src/main/java/org/openhab/habdroid/util/MjpegInputStream.kt

```diff
@@ -21,6 +21,7 @@ import java.io.ByteArrayInputStream
 import java.io.DataInputStream
 import java.io.IOException
 import java.io.InputStream
+import java.lang.IllegalArgumentException
```

Comment:
This import seems unneeded to me.

Reply:
Why? `IllegalArgumentException` is catched below and all other exceptions here are also imported.

Reply:
Because `java.lang` imports happen implicitly anyway. You don't import `java.lang.String` either ;-)